ast/Python.asdl

```diff
@@ -10,20 +10,22 @@ module Python
 
     stmt = FunctionDef(identifier name, arguments args,
                        stmt* body, expr* decorator_list, expr? returns,
-                       string? type_comment)
+                       string? type_comment, type_param* type_params)
```

Comment:
Unrelated to this PR: @youknowone do you know if RustPython parses out `type_comment`s or if we could remove this (unused) field?

Reply:
We don't parse it now. So removing it is okay right now.
To let RustPython keep the field for AST, changing to the type to empty tuple `()` will be better than entirely removing it.

Since type_comment is a part of Python3 spec, I thought it is expected to be implemented. CPython parser has an option flag to parse it or not.